pdg/src/snapshots/c2rust_pdg__tests__analysis_test_pdg_snapshot.snap

```diff
@@ -3,999 +3,999 @@ source: pdg/src/main.rs
 expression: pdg
 ---
 g {
-	n[0]: &_5  _    => _10 @ bb5[4]: fn main;
-	n[1]: copy n[0] => _9  @ bb5[5]: fn main;
-	n[2]: copy n[0] => _9  @ bb5[5]: fn main;
+	n[0]: &_5  _    => _10 @ bb5[4]: fn main; _10 = &mut _5   
+	n[1]: copy n[0] => _9  @ bb5[5]: fn main; _9 = &mut (*_10)
+	n[2]: copy n[0] => _9  @ bb5[5]: fn main; _9 = &mut (*_10)
 }
 nodes_that_need_write = []
 
 g {
-	n[0]: &_20 _    => _19 @ bb8[11]: fn main;  
-	n[1]: copy n[0] => _18 @ bb8[12]: fn main;  
-	n[2]: copy n[1] => _17 @ bb8[13]: fn main;  
-	n[3]: copy n[2] => _1  @ bb0[0]:  fn new_v1;
+	n[0]: &_20 _    => _19 @ bb8[11]: fn main;   _19 = &_20                                           
+	n[1]: copy n[0] => _18 @ bb8[12]: fn main;   _18 = &(*_19)                                        
+	n[2]: copy n[1] => _17 @ bb8[13]: fn main;   _17 = move _18 as &[&str] (Pointer(Unsize))          
+	n[3]: copy n[2] => _1  @ bb0[0]:  fn new_v1; _16 = std::fmt::Arguments::new_v1(move _17, move _21)
 }
 nodes_that_need_write = []
 
 g {
-	n[0]: &_13 _    => _27 @ bb8[22]: fn main;     
-	n[1]: copy n[0] => _26 @ bb8[23]: fn main;     
-	n[2]: copy n[1] => _1  @ bb0[0]:  fn new_debug;
+	n[0]: &_13 _    => _27 @ bb8[22]: fn main;      _27 = &_13                                                            
+	n[1]: copy n[0] => _26 @ bb8[23]: fn main;      _26 = &(*_27)                                                         
+	n[2]: copy n[1] => _1  @ bb0[0]:  fn new_debug; _25 = std::fmt::ArgumentV1::new_debug::<std::string::String>(move _26)
 }
 nodes_that_need_write = []
 
 g {
-	n[0]: &_24 _    => _23 @ bb13[4]: fn main;  
-	n[1]: copy n[0] => _22 @ bb13[5]: fn main;  
-	n[2]: copy n[1] => _21 @ bb13[6]: fn main;  
-	n[3]: copy n[2] => _2  @ bb0[0]:  fn new_v1;
+	n[0]: &_24 _    => _23 @ bb13[4]: fn main;   _23 = &_24                                                 
+	n[1]: copy n[0] => _22 @ bb13[5]: fn main;   _22 = &(*_23)                                              
+	n[2]: copy n[1] => _21 @ bb13[6]: fn main;   _21 = move _22 as &[std::fmt::ArgumentV1] (Pointer(Unsize))
+	n[3]: copy n[2] => _2  @ bb0[0]:  fn new_v1; _16 = std::fmt::Arguments::new_v1(move _17, move _21)      
 }
 nodes_that_need_write = []
 
 g {
-	n[0]: &_1 _ => _29 @ bb15[11]: fn main;
+	n[0]: &_1 _ => _29 @ bb15[11]: fn main; _29 = &mut _1
 }
 nodes_that_need_write = []
 
 g {
-	n[0]: copy _    => _34 @ bb16[5]: fn main;  
-	n[1]: copy n[0] => _2  @ bb0[0]:  fn expect;
+	n[0]: copy _    => _34 @ bb16[5]: fn main;   _34 = &(*_35)                                                                                 
+	n[1]: copy n[0] => _2  @ bb0[0]:  fn expect; _31 = std::result::Result::<std::ffi::CString, std::ffi::NulError>::expect(move _32, move _34)
```

Comment:
Can we prune paths like `rustc` error messages do, especially for types in the prelude?  I'm not sure how that's done, but maybe there's an exposed function in the `rustc` crates.